packages/icons/src/Icon.js

```diff
@@ -5,14 +5,22 @@ import PropTypes from 'prop-types'
 import * as icons from './index'
 
 const BaseIcon = ({ name, title, desc, titleId, descId, ...props }) => {
+  const isDev = process.env.NODE_ENV !== 'production'
+
+  if (isDev) {
+    console.warn(
```

Comment:
since `isDev`, maybe `console.trace` for a better stack trace?

Reply:
Somehow I didn't know `console.trace` was a thing 🤯